static/app/views/organizationGroupDetails/groupEvents.spec.jsx

```diff
@@ -47,6 +47,14 @@ describe('groupEvents', function () {
     });
 
     browserHistory.push = jest.fn();
+
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/events/',
+      body: {
+        data: [],
```

Comment:
If you want to add one more test w/ content, an easy way of adding a better mock is to just copy the result from the api response in your network tab here and simplify it a bit.

Reply:
Added data to the test, its also likely that i'll add more test cases in the future as well as things move forwards